src/core/url_store/encoder.ts

```
@@ -20,11 +20,14 @@ export const urlEncoder = new URLDataInSearchEncoder({
       encode: (position: MapPosition): string => {
         const [zoom, lat, lng] = position;
         const precision = Math.max(3, Math.ceil(Math.log(zoom) / Math.LN2));
+        const formatCoordinate = (n: number) => {
+          const fixedPointNumber = Number(n.toFixed(precision));
+          return (fixedPointNumber === -0 ? 0 : fixedPointNumber).toFixed(precision);
```

Biome (1.9.4)

[error] Do not use the === operator to compare against -0.

Safe fix: Replace -0 with 0


(lint/suspicious/noCompareNegZero)